demo/basic/templates/k8suniquelabels_template.yaml

```diff
@@ -8,7 +8,7 @@ spec:
       names:
         kind: K8sUniqueLabel
         listKind: K8sUniqueLabelList
-        plural: k8suniquelabels
+        plural: k8suniquelabel
```

Comment:
https://github.com/open-policy-agent/gatekeeper/blob/94e25f8b8370bd7df312833aedbf9dfcdd8dc371/demo/basic/templates/k8suniquelabels_template.yaml#L4

Reply:
I think this should be updated to `k8suniquelabel `

Reply:
Good catch, fixed